README.md

```diff
@@ -1,55 +1,45 @@
 
-## Resources & Library Documentation:
+## **Running Mission Control for those not working on Mission Control**
 
-If unfamiliar with Vue and Typescript here are some resources:
+Follow the steps below based on your role and setup:
 
-- [Official Vue Tutorial](https://vuejs.org/tutorial/#step-1)
-- [Typescript Documentation](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-oop.html) -
-- There is not a lot of documentation/tutorial on RosBridge and Roslib library but here is a collection of resources:
-  [RosBridge Protocol](https://github.com/RobotWebTools/rosbridge_suite/blob/ros1/ROSBRIDGE_PROTOCOL.md)
-- Overview: [https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp](https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp)
-- [Roslibjs source code]: https://github.com/RobotWebTools/roslibjs
+### **For Software Members not working on mission control**
 
-## Recommended IDE Setup
+- **Testing Locally (on personal Docker):**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the URC instructions to run the Docker container. Click on the Navbar websocket icon to switch to local development
+- **Running on the Rover:**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the **Launching on Rover** instructions below.
 
-- [VSCode](https://code.visualstudio.com/)
+### **For Non-Software Members**
 
-- [Vue DevTools Chrome Extension](https://devtools.vuejs.org/getting-started/installation): Can Inspect Components State and integrated with Pinia, allowing to see store's state
+Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the **Launching on Rover** instructions below.
 
-## Project Setup
+## **Launching on Rover**
 
-1. Install Node.js version 22.11.0 (LTS) (Long-term support Updated 11/6/2024)
-   - **Windows and macOS**:
-   1. If different node version is already installed, delete it from your system or if using version manager, switch to it.
-      - **Windows**: https://stackoverflow.com/questions/20711240/how-to-completely-remove-node-js-from-windows (Tested 11/6/2024)
-   2. [Download version 22.11.0 (LTS)](https://nodejs.org/en/download/prebuilt-installer) and run the installer
-   - **Linux**: [Follow Instructions to install 22.11.0 (LTS)](https://nodejs.org/en/download/package-manager) (Not tested)
-2. Run `git clone https://github.com/TrickfireRobotics/mission-control.git` in terminal **(not in the URC Docker Container)**
-3. Run `cd mission-control` to open the repository
-4. Run `npm install` to install dependencies
+Setup router and ensure connect to trickfirerouter on your computer. The router password is on the router itself Turn the rover computer on by plugging in the c port charger. You will know the computer is on if the LED lights up.
+Run the commands below in Powershell or in your OS terminal. When prompted for a password, enter `trickfire`
```

Comment:
I know it isn't exactly the most secure password, but should we really be putting it in our public GitHub? I guess it's already here now.